doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -664,7 +617,7 @@ Passthrough a Hard Disk to RTVM
 
       passthru_vpid=(
       ["eth"]="8086 156f"
-      ["sata"]="8086 9dd3"
+      ["sata"]="8086 a0d3"
```

Comment:
Thanks for updating this one. You also need to update the `lspci` command above though because they do not match now.

Reply:
Updated.